Automattic/newspack-newsletters
# Content validation: warn about nested groups

### All Submissions:






### Changes proposed in this Pull Request:



Adds a warning if there are non-top-level groups in the content. Such cases are not handled currently.

### How to test the changes in this Pull Request:

1. Create a newsletter with a nested group (e.g. a group in a column)
2. Observe that a warning notice is rendered above the block.

### Other information:
